### PR TITLE
Simplified extent

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Simplified extent by converting from Date to number at the start of the 'extent' logic, then converting back at the end.

### CODE REVIEW

**Comment by chrisprice** [comment]
My expectation was that there would be no need to treat dates any
differently apart from wrapping them at the end i.e. no getTime calls. Did
that not work?

File: src/util/extent.js [hunk 1]
```diff
@@ -51,49 +51,27 @@ export default function() {
...
+            var symmertical = dateExtent ? symmetricalAbout.getTime() : symmetricalAbout;
+            var distanceFromMax = Math.abs(max - symmertical),
+                distanceFromMin = Math.abs(min - symmertical),
+                halfRange = Math.max(distanceFromMax, distanceFromMin);
 
-            min = new Date(oldMin - delta);
-            max = new Date(oldMax + delta);
+            min = symmertical - halfRange;
+            max = symmertical + halfRange;
         }
```
Comment (owennw) [inline src/util/extent.js]: There's a typo in the variable name `symmertical`.

Comment (ColinEberhardt) [inline src/util/extent.js]: well spotted - thanks! fixed'n'squashed

**Comment by ColinEberhardt** [comment]
> My expectation was that there would

No, because ... WAT?!

**Pull request merged**